ghproxy/ghmetrics/ghpath.go

```diff
@@ -0,0 +1,424 @@
+package ghmetrics
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+)
+
+func getSimplifiedPath(path string) string {
```

Comment:
A single comment on this function about what it does would go a long way. It would also communicate that this is the entry function to this file.

Turning a GitHub API path into a genericized version for logging is very useful, and you can communicate that without forcing someone to grok the tree-implementation here.

Reply:
Optionally, rename to `GetSimplifiedPath`, even if you don't plan on reusing it. That would also indicate that this is the entry point.